backend/WikiContrib/Install.md

```diff
@@ -6,6 +6,16 @@ The present doc deals about installing and running `backend`. If you go inside `
 
 # Steps to setup server locally
 
+## Using Docker 
+
+1. Install Docker in your system (https://docs.docker.com/install/)
```

Comment:
Maybe for this step in both `backend/../Install.md` & `frontend/../Install.md/` add something like:
* One should ensure that the Docker has been successfully installed and only then should proceed. 
* This step should be executed once for backend & frontend.